uint32 sequence_id
uint8 exec_level
uint8 feedback_level
string instruction
string[] string_values
float32[] float_values

